010_Intro/15_API.asciidoc

```diff
@@ -30,7 +30,7 @@ otherwise, they may not be able to understand each other.
 ====
 
 More information about the Java clients can be found in the Java API section
-of the http://www.elasticsearch.org/guide/[Guide].
+of the http://www.elastic.co/guide/[Elastic documentation].
```

Comment:
It's now in the clients section, perhaps link to https://www.elastic.co/guide/en/elasticsearch/client/index.html instead?